pkg/tfgen/installation_docs_test.go

```diff
@@ -184,3 +189,83 @@ func TestApplyEditRules(t *testing.T) {
sult;\n```\n{{% /choosable %}}\n{{% choosable language python %}}\n```yaml\n# Pulumi.yaml provider configuration file\nname: configuration-example\nruntime: python\n\n```\n```python\nimport pulumi\nimport pulumi_simple as simple\n\na_resource = simple.index.Resource(\"a_resource\",\n    renamed_input1=hello,\n    input_two=True)\npulumi.export(\"someOutput\", a_resource[\"result\"])\n```\n{{% /choosable %}}\n{{% choosable language csharp %}}\n```yaml\n# Pulumi.yaml provider configuration file\nname: configuration-example\nruntime: dotnet\n\n```\n```csharp\nusing System.Collections.Generic;\nusing System.Linq;\nusing Pulumi;\nusing Simple = Pulumi.Simple;\n\nreturn await Deployment.RunAsync(() => \n{\n    var aResource = new Simple.Index.Resource(\"a_resource\", new()\n    {\n        RenamedInput1 = \"hello\",\n        InputTwo = true,\n    });\n\n    return new Dictionary<string, object?>\n    {\n        [\"someOutput\"] = aResource.Result,\n    };\n});\n```\n{{% /choosable %}}\n{{% choosable language go %}}\n```yaml\n# Pulumi.yaml provider configuration file\nname: configuration-example\nruntime: go\n\n```\n```go\npackage main\n\nimport (\n\t\"github.com/pulumi/pulumi-simple/sdk/go/simple\"\n\t\"github.com/pulumi/pulumi/sdk/v3/go/pulumi\"\n)\n\nfunc main() {\n\tpulumi.Run(func(ctx *pulumi.Context) error {\n\t\taResource, err := simple.NewResource(ctx, \"a_resource\", &simple.ResourceArgs{\n\t\t\tRenamedInput1: \"hello\",\n\t\t\tInputTwo:      true,\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\tctx.Export(\"someOutput\", aResource.Result)\n\t\treturn nil\n\t})\n}\n```\n{{% /choosable %}}\n{{% choosable language yaml %}}\n```yaml\n# Pulumi.yaml provider configuration file\nname: configuration-example\nruntime: yaml\n\n```\n```yaml\nresources:\n  aResource:\n    type: simple:resource\n    name: a_resource\n    properties:\n      renamedInput1: hello\n      inputTwo: true\noutputs:\n  someOutput: ${aResource.result}\n```\n{{% /choosable %}}\n{{% choosable language java %}}\n```yaml\n# Pulumi.yaml provider configuration file\nname: configuration-example\nruntime: java\n\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.simple.resource;\nimport com.pulumi.simple.ResourceArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        var aResource = new Resource(\"aResource\", ResourceArgs.builder()\n            .renamedInput1(\"hello\")\n            .inputTwo(true)\n            .build());\n\n        ctx.export(\"someOutput\", aResource.result());\n    }\n}\n```\n{{% /choosable %}}\n{{< /chooser >}}\n\n\n## Configuration Reference\n\nThe following configuration inputs are supported:",
```

Comment:
Could we use hexops/autogold to make this multiline and readable?

Reply:
Unfortunately, autogold flails here too, because the string in question contains back ticks. I've created files the represent the input and expected transformation instead for improved readability.